app/views/organisations/_show_organisation.html.erb

```diff
@@ -130,6 +130,10 @@
   <% end %>
 <% end %>
 
+<% unless @organisation.is_promotional_org? %>
+  <%= render partial: 'foi' %>
```

Comment:
I think calling the partial `freedom_of_information` (or similar) may be clearer - wouldn't want to assume someone knows what FOI means.